### DISCUSSION TITLE
Can't buy the pro version because The VAT verification feature doesn't work

### DISCUSSION THREAD

**Discussion by nomad5000** [Laravel-Backpack/community-forum#424] [Q&A (Help)] [answered]
Try to buy pro version with VAT number -> keep getting error of VIS Service not working. After that I can't pay.

Please check.

**Comment by nomad5000** [comment]
If you provide an email for your orders support I can contact them directly instead.

**Reply by pxpm** [reply]
Hey @nomad5000 sorry for the bad experience. 

Please email `hello@backpackforlaravel.com` with your inquiry. We will investigate the issue. 

Thanks for letting us know 🙏

**Comment by tabacitu** [comment, marked as answer]
Hi @nomad5000 ,

We use the official EU API to check the VAT number, and that API was down. But **it's back up now. You should be able to purchase without a hitch.** We've discovered the API is pretty unreliable, it's down for a few minutes every few months so... we are investigating replacing it with a third-party (non-official), to prevent these problems in the future.

Sorry for the bad experience. 
Cheers!
